app/Commands/EditEnvironmentVariables.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace App\Commands;
+
+use App\Helpers\Helpers;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+class EditEnvironmentVariables extends Command
+{
+    /**
+     * The signature of the command.
+     *
+     * @var string
+     */
+    protected $signature = 'aws:edit-environment-variables';
+
+    /**
+     * The description of the command.
+     *
+     * @var string
+     */
+    protected $description = 'Edits environment variables for a system';
+
+    /** @var Helpers $helpers */
+    protected $helpers;
+
+    protected $clusterName;
+    protected $serviceName;
+    protected $taskDefinitionName;
+
+    private array $envFiles;
+
+    private string $fileName;
+
+    private string $fileContents;
+
+    public function __construct(Helpers $helpers)
+    {
+        $this->helpers = $helpers;
+        parent::__construct();
+    }
+
+    public function configure()
+    {
+        $this->setDefinition(array_merge([
+            new InputOption('client-system', null, InputOption::VALUE_OPTIONAL, 'The client to edit the environment variables for'),
```

Comment:
Not sure `client-system` is the best name for this? From what I gather reading the code, it expects the s3 bucket name?

Perhaps just `s3-bucket-name`?

Reply:
Done